WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -89,5 +94,5 @@
     }
   });
 
-  $('#<%= params[:display] == "map" ? "display-map" : "display-list" %>').button('toggle');
+  $('#display-<%= params[:display] %>').button('toggle');
```

Comment:
This is opens us up to an injection attack, as anyone can put whatever they want in the display parameter. I believe @jonatanklosko ran into this same issue in a past PR.

Unfortunately, the way the code is witten right now, it's very tempting for a clever coder to come by and clean this up as you just did. I think we should add a comment here explaining why this is written the way it is.